components/Eingabefeld 1/index.test.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
```

Comment:
Please check the naming of the folder as well: `Eingabefeld 1` as a folder name where the `index.js` contains a component called `InformationPage` is not consistent.

You don't have any inputs here, so `Eingabefeld` is misleading. What about `components/Information` and the component can be called `InformationDisplay` os something similar?

I would not recommend to call the component something `...Page` because it is not a page in the `pages` folder.